.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
https://git-scm.com/docs/git-config#Documentation/git-config.txt-coreexcludesFile
This is the configuration value you want, to be able to have a global ignore file to put in things like `.vscode`.